### ISSUE TITLE
Keyboard focus and selected/hovered row are not visible when applying 'System high contrast' themes.

### ISSUE THREAD

**Issue by alonzrihan** [ceolter/ag-grid#9132]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => see 'Providing a Reproducible Scenario'
```

**Providing a Reproducible Scenario**
- On Windows device, navigate to Settings> Accessibility> Contrast themes.
- Apply any of the following themes: Aquatic, Desert, Dusk and Night sky.
- Hover over a row and see that there is no indication for the row or the focused cell

**Current behavior**
There is no indication for focused cell or hovered row
![image](https://github.com/user-attachments/assets/f8f47e22-5834-4809-b9e9-6749082d4943)
![image](https://github.com/user-attachments/assets/0e5cba27-790b-4432-813a-2705849bebf8)
![image](https://github.com/user-attachments/assets/07a0ebf3-ba72-4ee0-b55c-e4361f76f33f)
![image](https://github.com/user-attachments/assets/16aaae6d-a39b-427f-9eff-0934f5cd776b)

**Expected behavior**
Show focused cell and row, like in "normal" mode:
![image](https://github.com/user-attachments/assets/58f7edff-0136-4c77-be9d-64f0fc06cb4b)

**Please tell us about your environment:**
- Windows
- React

* **AG Grid version:** 32.0.0
Issue still exists in latested version (the one previewed in your website)

* **Browser:**
- Chrome
- Edge

* **Language:** [TypeScript 5.6.2]

**Comment by AG-Zoheil** [comment]
Hello 

Thank you for reporting this - much appreciated!

Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference: **AG-6163**

We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.

You can follow the status of this item on the product pipeline page:
https://ag-grid.com/pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil
